### PR TITLE
week 13 api-project

### PR DESCRIPTION
**Pull request by sansan-sakura**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
-## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+If I could do it again, I would work more with error handlings. As my knowdlege of Node.js/express is not much, I could not do a lot of them. So definetly, I would like to add that in the future.
 
-## View it live
-
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+API:https://my-first-api-91k4.onrender.com/
+Deployed site:https://cosmic-strudel-6f1e10.netlify.app/
```
Comment (mikaeloling) [inline README.md:16]: Please check this site, for me it did not work. (API works, Deployed site not working)

File: frontend/src/components/Card.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { NobelRes } from "../types/type";
+
+export const Card = ({ nobel }: { nobel: NobelRes }) => {
```
Comment (mikaeloling) [inline frontend/src/components/Card.tsx:3]: This looks very impressive code-wise, also interesting that you chose to work with typescript!

**Review by mikaeloling — APPROVED** [review]
I really look forward to seeing the deployed site once you get this one working: https://cosmic-strudel-6f1e10.netlify.app/. The API is working and have met all the requirements.

File: README.md (hunk 1 above)
Comment (sansan-sakura) [reply README.md:16]: Thank you for letting me know!! I put the wrong one!

**Pull request closed without merging**